cmd/kube-proxy/app/server.go

```diff
@@ -518,7 +520,9 @@ with the apiserver API to configure the proxy.`,
 		os.Exit(1)
 	}
 
-	opts.AddFlags(cmd.Flags())
+	fs := cmd.Flags()
+	opts.AddFlags(fs)
+	fs.AddGoFlagSet(goflag.CommandLine) // for --boot-id-file and --machine-id-file
```

Comment:
There was `pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)` in `cmd/kube-proxy/proxy.go` which copied `boot-id-file` and `machine-id-file`. It made more sense to me to move it here and have the same, simple `main` as in other commands.

That's okay because this particular `NewProxyCommand` is internal and not used by downstream, so we can change which flags it adds.

Reply:
I have no idea what `boot-id-file` and `machine-id-file` are - `git grep` shows nothing?

Reply:
Oh!  They are defined with underscores and are part of cadvisor.  BLECCHCHHH